amp stack rm command panics when no argument is passed

`amp stack rm` command panics when no argument is passed

```
$ amp stack rm
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/appcelerator/amp/cli/command/stack.NewRemoveCommand.func1(0xc420422000, 0xc4203e5340, 0x0, 0x2, 0x0, 0x0)
	/Users/nviswanathan/go/src/github.com/appcelerator/amp/cli/command/stack/remove.go:28 +0xac
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).execute(0xc420422000, 0xc4203e5160, 0x2, 0x2, 0xc420422000, 0xc4203e5160)
	/Users/nviswanathan/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:643 +0x3ef
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420416000, 0xc4203e4820, 0xc4203e4808, 0xc4203c1eb0)
	/Users/nviswanathan/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:734 +0x339
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420416000, 0xc420373ea0, 0xc420416000)
	/Users/nviswanathan/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:692 +0x2b
main.main()
	/Users/nviswanathan/go/src/github.com/appcelerator/amp/cmd/amp/main.go:35 +0x157
```

--- a/cli/command/stack/remove.go
+++ b/cli/command/stack/remove.go
@@ -24,6 +24,7 @@ func NewRemoveCommand(c cli.Interface) *cobra.Command {
 		Use:     "remove STACKNAME",
 		Aliases: []string{"rm", "down", "stop"},
 		Short:   "Remove a deployed stack",
+		PreRunE: cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ropts.name = args[0]
 			return remove(c)

